deploy/flux-deployment.yaml

```diff
@@ -13,7 +13,7 @@ spec:
   template:
     metadata:
       annotations:
-        prometheus.io/port: "3031" # tell prometheus to scrape /metrics endpoint's port.
+        prometheus.io.port: "3031" # tell prometheus to scrape /metrics endpoint's port.
```

Comment:
What the heck? The template has a slash.

Reply:
Ahhh, this happened during rebasing. I didn't incorporate the changes made on deploy/